.gitignore

```diff
@@ -19,3 +19,4 @@ tmp
 .rvmrc
 _site
 .sass-cache
+file::memory:?cache=shared
```

Comment:
What, why? I see you were trying to test this on in-memory db, but it created a file instead, right?